changelog/__init__.py

```diff
@@ -21,6 +21,12 @@
 Commit = namedtuple('Commit', ['sha', 'message'])
 PullRequest = namedtuple('PullRequest', ['number', 'title'])
 
+# Merge commits use a double linebreak between the branch name and the title
+MERGE_PR_RE = re.compile(r'^Merge pull request #([0-9]+) from .*\n\n(.*)')
+
+# Squash-and-merge commits use the PR title with the number in parentheses
+SQUASH_PR_RE = re.compile(r'^(.*) \(#([0-9]+)\)\n\n.*')
```

Comment:
::sigh:: You need to remove the double linebreak from this. If a squash-n-merge commit only rolls up a single commit, there is nothing after the PR number. See: https://github.com/cfpb/cfgov-refresh/commit/e814f8f394029c96fdab9ed1e690ad97797d58fd which does not appear in [the latest release's changelog](https://github.com/cfpb/cfgov-refresh/releases/tag/5.0.5).